response/ach.go

```diff
@@ -0,0 +1,7 @@
+package response
+
+type GetAchPull struct {
```

Comment:
Let's call this type `ACHPull` instead

Reply:
Fixed: https://github.com/dapi-co/dapi-go/pull/9/commits/e43302f6280ff851ac19283223a685b1e090eec7